bokeh.oggm.org/docker-compose.yml

```diff
@@ -84,17 +84,19 @@ services:
       - "traefik.http.routers.hma_glacial_lakes-app.entrypoints=web"
       - "traefik.http.routers.hma_glacial_lakes-app.rule=PathPrefix(`/hma_glacial_lakes`)"
 
-  dtcg_l2_dashboard-app:
+  dtcgweb-app:
     image: "ghcr.io/oggm/bokeh:20250915"
-    container_name: "dtcg_l2_dashboard-app"
-    command:
-      - "git+https://github.com/DTC-Glaciers/dtcg-web.git@v0.1.2"
-      - "src/dtcgweb/main.py"
+    container_name: "dtcgweb-app"
+    build:
+      context: https://github.com/DTC-Glaciers/dtcg-web.git#v0.1.3
+      args:
+        - "BOKEH_PREFIX=/dtcgweb"
+        - "BOKEH_ALLOW_WS_ORIGIN=${WS_ORIGIN}"
```

Comment:
I don't see these being used anywhere in the Dockerfile, so these likely don't do anything?